src/main/kotlin/org/jetbrains/bio/experiments/fit/Span2PeakCallingExperiment.kt

```diff
@@ -0,0 +1,169 @@
+package org.jetbrains.bio.experiments.fit
+
+import org.jetbrains.bio.big.BigWigFile
+import org.jetbrains.bio.coverage.Coverage
+import org.jetbrains.bio.coverage.Fragment
+import org.jetbrains.bio.dataframe.DataFrame
+import org.jetbrains.bio.genome.Chromosome
+import org.jetbrains.bio.genome.ChromosomeRange
+import org.jetbrains.bio.genome.GenomeQuery
+import org.jetbrains.bio.query.CachingQuery
+import org.jetbrains.bio.query.ReadsQuery
+import org.jetbrains.bio.query.reduceIds
+import org.jetbrains.bio.query.stemGz
+import org.jetbrains.bio.statistics.ClassificationModel
+import org.jetbrains.bio.statistics.Fitter
+import org.jetbrains.bio.statistics.hypothesis.NullHypothesis
+import org.jetbrains.bio.statistics.mixture.PoissonRegressionMixture
+import org.jetbrains.bio.statistics.state.ZLH
+import org.jetbrains.bio.util.div
+import org.jetbrains.bio.viktor.asF64Array
+import java.nio.file.Path
+
+class Span2PeakCallingExperiment<Model : ClassificationModel, State : Any>(
```

Comment:
Please add some comments describing the difference between `SpanPeakCallingExperiment` and `Span2PeakCallingExperiment` with references to the first version.

Reply:
Descriptions added.